AsyncDisplayKit/Private/ASInternalHelpers.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <UIKit/UIKit.h>
+#import "ASBaseDefines.h"
+
+@class ASLayoutChild;
```

Comment:
Thanks for creating ASInternalHelpers.h and ASLayoutSpecUtilities.h!  You can probably remove this ASLayoutChild declaration, and do we need the UIKit import?